xml/System.Linq/Enumerable.xml

```diff
@@ -1694,9 +1698,21 @@ from int i in objects
 ```vb  
 From i As Integer In objects  
 ```  
-  
-   
-  
+
+The `select` clause of a query enables other conversion types, like the implicit numeric conversions. The following example shows using both the `Cast` method and a `select` statement to convert a sequence of boxed integers to a sequence of doubles.
```

Comment:
Alternative for the first sentence (to be more inline with the style guide):
Use `select` clause of a query to perform other conversion types, like the implicit numeric conversions.